tests/test_resolve.py

```
@@ -505,7 +506,13 @@ def test_set_frontmatter_field_refuses_a_readonly_spec(tmp_path):
 
 
 def test_build_context_gathers_critical_escalations(tmp_path):
-    run_dir, state, task = _escalated_run(tmp_path, spec_file="/abs/spec.md")
+    # An absolute `spec_file` passes through `runs.task_spec_path` verbatim. The literal
+    # has to be OS-absolute, not merely rooted: on Windows "/abs/spec.md" is DRIVE-relative
+    # (`Path.is_absolute()` is False), so it takes the anchoring arm instead and pathlib's
+    # `/` keeps the root's drive while discarding its path — yielding "D:/abs/spec.md", a
+    # shape no real run persists. `tmp_path` is absolute on every OS.
+    spec = tmp_path / "abs" / "spec.md"
+    run_dir, state, task = _escalated_run(tmp_path, spec_file=str(spec))
```

Ruff (0.16.2)

[warning] Unpacked variable `task` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)